Tests/PackageDescription4Tests/PackageTests.swift

```diff
@@ -41,12 +41,33 @@ class PackageTests: XCTestCase {
             cLanguageStandard: .c99,
             cxxLanguageStandard: .cxx14
         )
-        XCTAssertEqual(package.toJSON().toString(), """
-            {"cLanguageStandard": "c99", "cxxLanguageStandard": "c++14", "dependencies": [{"requirement": {"lowerBound": "1.0.0", "type": "range", "upperBound": "2.0.0"}, "url": "/dep1"}, {"requirement": {"identifier": "1.0.0", "type": "exact"}, "url": "/dep2"}, {"requirement": {"lowerBound": "1.0.0", "type": "range", "upperBound": "2.1.2"}, "url": "/dep3"}, {"requirement": {"identifier": "ref", "type": "revision"}, "url": "/dep4"}, {"requirement": {"identifier": "master", "type": "branch"}, "url": "/dep5"}], "name": "Foo", "products": [{"name": "foo", "product_type": "library", "targets": ["foo"], "type": null}, {"name": "bar", "product_type": "library", "targets": ["bar"], "type": "dynamic"}, {"name": "exec", "product_type": "executable", "targets": ["exe"]}], "swiftLanguageVersions": [3, 4], "targets": [{"dependencies": [{"name": "dep1", "type": "byname"}, {"name": "dep2", "package": null, "type": "product"}, {"name": "bar", "type": "target"}], "exclude": [], "isTest": false, "name": "foo", "path": null, "publicHeadersPath": null, "sources": null}, {"dependencies": [], "exclude": [], "isTest": false, "name": "bar", "path": null, "publicHeadersPath": null, "sources": null}, {"dependencies": [{"name": "foo", "type": "byname"}, {"name": "bar", "type": "byname"}], "exclude": [], "isTest": true, "name": "allTests", "path": null, "publicHeadersPath": null, "sources": null}]}
+
+        let json: String
+        do {
+            let encoder = PackageDescription4.JSONEncoder()
+            json = try encoder.encode(package)
```

Comment:
If you make the test throws then you need to do this do catch stuff right? The test runner will catch the error and report it failed and print the error

Reply:
True, but the displayed error is less than optimal. I'm doing this to get the `String(describing:)` version of the error. But perhaps I shouldn't be doing this. @aciidb0mb3r what do you think?